requirements/product-environment/product-environment.tex

```diff
@@ -0,0 +1,41 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}  
+      \begin{itemize}
+      \item A connection to a LAN with a central server
+      \item 2 MB of storage space and additional 100MB per image
+      \end{itemize}
+    \subsubsection{Software Requirements}  
+      \begin{itemize}
+      \item None, so far
+    \end{itemize}
+
+  \subsection{Central Server}
+    \subsubsection{Hardware Requirements}  
+    \begin{itemize}
+      \item A connection to a LAN
+      \item 500 MB of storage space
+      \item 8 GB of RAM
```

Comment:
I highly doubt that we will need 8 GB of RAM.
We could probably make do with just 1-2 GB if we use a private docker hub to store queued jobs.

Reply:
I used [this](https://docs.docker.com/v17.09/datacenter/ucp/2.1/guides/admin/install/system-requirements/) to determine the hardware requirements, I assumed that the central server is a manager node. Still, I'll change it to 2 GB for now.